.github/workflows/run-tests.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [ 8.x, 10.x, 11.x ]
+        node-version: [ 12.x, 13.x, 14.x, 15.x ]
```

Comment:
Same as above. It would be better to have a ticket for upgrading the node-versions and have this in a separate PR after we test for publishing.

I remember there were some dependency issues when I played around with different node versions.

Reply:
Same as above.